releasenotes/notes/truncate-long-span-attributes-a4900ee9844160b4.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: Fixes an issue where span attributes were not truncated before encoding, leading to runtime error and causing spans to be dropped. 
+    Spans with resource name, tag key or value larger than 25000 characters will be truncated to 2500 characters. 
```

Comment:
nit: worth adding that this only allows the python tracer to submit the data onwards, but strings are still subject to the guidelines listed in https://docs.datadoghq.com/tracing/troubleshooting/?tab=python#data-volume-guidelines